src/frontend/src/components/Landing/index.tsx

```diff
@@ -70,113 +76,369 @@ class Landing extends React.Component {
                   </h2>
-                  <p className="lh-copy mb3">
-                    We only ask that you <a href="http://eepurl.com/g6N3Bn" className="link bb hover-lightest-blue">provide your contact details</a> and obtain OECI credentials.
-                  </p>
-                  <p className="mb4">
-                    <Link to="/manual#overview" className="link bb hover-lightest-blue">
-                      Learn More
-                    </Link>
+                  <p className="white tc center mw6 mb4">
+                    Select a partner below near you. They can provide your analysis and help you file for expungement.
                   </p>
                 </div>
-                <div className="w-50-l pa4 pa5-l">
-                  <h2 className="f3 fw9 mb3">
-                    Are you looking to clear your&nbsp;record?
+                <div className="ba bw3 br3 b--lightest-blue1 bg-white mb6">
+                  <h3 className="f3 fw9 pa4">
+                    Partners
+                  </h3>
+                  <ul className="list">
+                    <li className="bt bw2 b--lightest-blue1">
+                    <Disclosure>
+                      <DisclosureButton className="flex-ns w-100 relative navy hover-blue pv3 ph3 ph4-ns">
+                        <span className="w-70 db fw7 pr3 mb2 mb0-ns">
+                          Portland Community College
+                        </span>
+                        <span className="w-30 pr3">
+                          Northeast Portland
+                        </span>
+                        <span className="absolute top-0 right-0 pt3 ph3">
+                          <span aria-hidden="true" className="fas fa-angle-down"></span>
+                        </span>
+                        </DisclosureButton>
+                        <DisclosurePanel>
+                      <div className="bl bw2 f5 b--blue pb3 ph3 ml3 ml4-ns" >
+                        Here I am! I am the buried treasure!
+                      </div></DisclosurePanel>
+                   </Disclosure> </li>
+                    
+                    <li className="bt bw2 b--lightest-blue1">
+                    <Disclosure>
+                      <DisclosureButton className="flex-ns w-100 relative navy hover-blue pv3 ph3 ph4-ns">
+                        <span className="w-70 db fw7 pr3 mb2 mb0-ns">
+                          Qiu-qiu Law
+                        </span>
+                        <span className="w-30 pr3">
+                          Portland
+                        </span>
+                        <span className="absolute top-0 right-0 pt3 ph3">
+                          <span aria-hidden="true" className="fas fa-angle-up"></span>
+                        </span>
+                      </DisclosureButton>
+                        <DisclosurePanel>
+                      <div className="bl bw2 f5 b--blue pb3 ph3 ml3 ml4-ns">
+                        <ul className="list mb3">
```

Comment:
This intendations should get corrected automatically by the git pre-commit hook. You should run `pre-commit install`. You may need to `pip install pre-commit` first.